templates/docs/examples/patterns/cta/block/default.html

```diff
@@ -0,0 +1,11 @@
+{% extends "_layouts/examples.html" %}
+{% block title %}CTA Block / Default{% endblock %}
+
+{% block standalone_css %}patterns_cta{% endblock %}
+
+{% block content %}
+<div class="p-cta-block">
+    <button class="p-button--positive">Learn more</button>
```

Comment:
@lyubomir-popov Should we expect that the link element is always an `a` tag? Would there ever be anything adjacent to the button that is not a link?

Reply:
How about using flex on `p-cta-block`, this way we don't have to target children at all.

<img width="456" alt="image" src="https://github.com/canonical/vanilla-framework/assets/83575/e31e7c14-a336-4362-899a-c058beaa1a4d">

I just wonder if there would be any unexpected behaviours because of that.

Reply:
Updated to use flex, let's see in QA and percy if it causes any issues.